src/script/util/avatarUtil.ts

```diff
@@ -0,0 +1,87 @@
+    // "| 0" ensures we keep it 32-bit integer.
+    // eslint-disable-next-line no-magic-numbers
+    hash = (31 * hash + str.charCodeAt(counter)) | 0;
+  }
+  return hash;
+}
+
+// colors for group avatar in dark and light theme
+const groupAvatarDarkThemeOptions = [
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.PURPLE_DARK_500, COLOR_V2.RED_DARK_500],
+  [COLOR_V2.BLUE_DARK_500, COLOR_V2.GREEN_DARK_500, COLOR_V2.AMBER_DARK_500],
+  [COLOR_V2.BLUE_DARK_500, COLOR_V2.RED_DARK_500, COLOR_V2.PURPLE_DARK_500],
+  [COLOR_V2.AMBER_DARK_500, COLOR_V2.GREEN_DARK_500, COLOR_V2.PURPLE_DARK_500],
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.PURPLE_DARK_500],
+  [COLOR_V2.BLUE_DARK_500, COLOR_V2.AMBER_DARK_500, COLOR_V2.GREEN_DARK_500],
+  [COLOR_V2.RED_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.PURPLE_DARK_500],
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.RED_DARK_500, COLOR_V2.BLUE_DARK_500],
+  [COLOR_V2.AMBER_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.PURPLE_DARK_500],
+  [COLOR_V2.PURPLE_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.GREEN_DARK_500],
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.PURPLE_DARK_500, COLOR_V2.RED_DARK_500],
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.RED_DARK_500],
+  [COLOR_V2.PURPLE_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.GREEN_DARK_500],
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.BLUE_DARK_500, COLOR_V2.AMBER_DARK_500],
+  [COLOR_V2.GREEN_DARK_500, COLOR_V2.AMBER_DARK_500, COLOR_V2.PURPLE_DARK_500],
+];
+
+const groupAvatarLightThemeOptions = [
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500, COLOR_V2.RED_LIGHT_500],
+  [COLOR_V2.BLUE_LIGHT_500, COLOR_V2.GREEN_LIGHT_500, COLOR_V2.AMBER_LIGHT_500],
+  [COLOR_V2.BLUE_LIGHT_500, COLOR_V2.RED_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500],
+  [COLOR_V2.AMBER_LIGHT_500, COLOR_V2.GREEN_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500],
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500],
+  [COLOR_V2.BLUE_LIGHT_500, COLOR_V2.AMBER_LIGHT_500, COLOR_V2.GREEN_LIGHT_500],
+  [COLOR_V2.RED_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500],
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.RED_LIGHT_500, COLOR_V2.BLUE_LIGHT_500],
+  [COLOR_V2.AMBER_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500],
+  [COLOR_V2.PURPLE_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.GREEN_LIGHT_500],
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500, COLOR_V2.RED_LIGHT_500],
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.RED_LIGHT_500],
+  [COLOR_V2.PURPLE_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.GREEN_LIGHT_500],
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.BLUE_LIGHT_500, COLOR_V2.AMBER_LIGHT_500],
+  [COLOR_V2.GREEN_LIGHT_500, COLOR_V2.AMBER_LIGHT_500, COLOR_V2.PURPLE_LIGHT_500],
+];
+
+/**
+ *
+ * @param id - unique id
+ * @param theme - default or dark
+ * @returns - array of colors for group avatar
+ */
+export function getGroupAvatarColors(
+  id = '',
+  theme: typeof PropertiesRepository.prototype.properties.settings.interface.theme,
```

Comment:
we have the accent-colors available as css variable that changes depending on theme, for example `var(--amber-500` is #a25915 in light mode and #ffd426 in dark mode.

Seems like we could use those instead of relying on the property repository to get access to the theme, WDYT?